bingosync-app/generators/dark_souls_generator.js

```diff
@@ -168,15 +166,14 @@ bingoList[16] = [
   { name: "Never use Consumables", types: ["restriction", "restrict_consumable"] },
   { name: "Only heal with Humanities", types: ["restriction", "restrict_consumable"] },
   { name: "Never use Pyromancy", types: ["restriction","restrict_spell"] },
-  { name: "Never wear Armour", types: ["restriction"] }
+  { name: "Never wear Armour", types: ["restriction","restrict_armour"] }
```

Comment:
Type added to try and enforce exclusivity between "never wear armour" and "starting armour only"